### PR TITLE
Ladybird: Accept file drops

### PR DESCRIPTION
**Pull request by krkk**

### CODE REVIEW

File: Ladybird/WebContentView.cpp [hunk 1]
```diff
@@ -322,6 +324,19 @@ void WebContentView::mouseReleaseEvent(QMouseEvent* event)
...
+void WebContentView::dragEnterEvent(QDragEnterEvent* event)
+{
+    if (event->mimeData()->hasUrls())
+        event->acceptProposedAction();
+}
+
+void WebContentView::dropEvent(QDropEvent* event)
+{
+    VERIFY(event->mimeData()->hasUrls());
+    load(ak_deprecated_string_from_qstring(event->mimeData()->urls()[0].toString()));
```
Comment (linusg) [inline Ladybird/WebContentView.cpp:336]: Wouldn't it make more sense to load the remaining URLs in a new tab? At least that's what Firefox does.

File: Ladybird/WebContentView.h [hunk 2]
```diff
@@ -68,7 +68,6 @@ class WebContentView final
     Function<void(const AK::URL&)> on_load_start;
     Function<void(const AK::URL&)> on_load_finish;
     Function<void(Gfx::Bitmap const&)> on_favicon_change;
-    Function<void(const AK::URL&)> on_url_drop;
```
Comment (linusg) [inline Ladybird/WebContentView.h:71]: Why is this removed?

Comment (krkk) [reply Ladybird/WebContentView.h:71]: it wasn't used at all (like most other Function<> members here) and it looks like it was copy-pasted from LibWebView.

<details>

https://github.com/SerenityOS/serenity/blob/aee7c44064b4ad24709e3b5c5b5bb63b3b2d7354/Userland/Libraries/LibWebView/OutOfProcessWebView.h#L83-L109

https://github.com/SerenityOS/serenity/blob/aee7c44064b4ad24709e3b5c5b5bb63b3b2d7354/Ladybird/WebContentView.h#L61-L83

</details>

so i just removed it to avoid confusion

Comment (linusg) [reply Ladybird/WebContentView.h:71]: I'd prefer if we deal with those separately instead of removing one of many unused ones now :)

File: Ladybird/Tab.cpp [hunk 3]
```diff
@@ -170,6 +159,12 @@ void Tab::navigate(QString url)
     view().load(ak_deprecated_string_from_qstring(url));
 }
 
+void Tab::load(URL const& url, LoadType load_type)
```
Comment (linusg) [inline Ladybird/Tab.cpp:162]: Now we have `navigate(url)` and `load(url, LoadType::HistoryNavigation)` and it's not at all clear what the difference is. Let's not do that.

File: Ladybird/BrowserWindow.cpp [hunk 4]
```diff
@@ -292,7 +295,7 @@ void BrowserWindow::debug_request(DeprecatedString const& request, DeprecatedStr
     m_current_tab->debug_request(request, argument);
 }
 
-void BrowserWindow::new_tab()
+void BrowserWindow::new_tab(URL const& url, bool activate)
```
Comment (linusg) [inline Ladybird/BrowserWindow.cpp:298]: Don't add positional boolean args, use an `Activate::Yes/No` enum

**Review by linusg — CHANGES_REQUESTED** [review]
Looks good, two small issues
